fleet_packages.json

```
@@ -57,5 +57,9 @@
   {
     "name": "security_detection_engine",
     "version": "9.0.6"
+  },
+  {
+    "name": "security_ai_prompts",
+    "version": "0.0.2"
```

You are probably missing the flag to allow prerelease packages, as done for APM package above:
```suggestion
    "name": "security_ai_prompts",
    "version": "0.0.2",
    "allowSyncToPrerelease": true
```